parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -253,17 +253,19 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
 fi
 {{end}}
 
+VALIDATION_ERR=0
+
 {{- if IsHostedMaster }}
```

Comment:
I think an easier way to complete the script no matter what is to just move this entire block down after the `ps auxfww >/opt/azure/provision-ps.log &` statement, using the original implementation.

Reply:
The only thing I'm worried about is reboot. If it happens to reboot, then checking connectivity would fail (or the process is gone?).

Reply:
think you could leave the reboot block and move the rest up. And even change it to reboot now.